javascript/js-rails/rails_backend.md

```diff
@@ -63,11 +65,11 @@ This section contains helpful links to other content. It isn't required, so cons
 
 #### With Ruby on Rails
 
-* Refresher -- how do you set up a basic Rails app?
-* Refresher -- how do you load custom Javascript in a given Rails view page?
-* How does "unobtrusive Javascript" work?
-* How can you pass data from your Rails app to your Javascript?
-* Why would you want to use AJAX to load large batches of data?
+* Refresher: <a class="knowledge-check-link" href="https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby-on-rails/lessons/your-first-rails-application-ruby-on-rails#your-first-rails-app" target="_blank"> How do you set up a basic Rails app? </a>
+* Refresher: <a class="knowledge-check-link" href="https://railsapps.github.io/rails-javascript-include-external.html#locations" target="_blank"> How do you load custom Javascript in a given Rails view page? </a>
+* <a class="knowledge-check-link" href="https://edgeguides.rubyonrails.org/working_with_javascript_in_rails.html#unobtrusive-javascript" target="_blank">How does "unobtrusive Javascript" work?</a>
+* <a class="knowledge-check-link" href="https://railsapps.github.io/rails-javascript-include-external.html#parameters" target="_blank">How can you pass data from your Rails app to your Javascript?</a>
+* <a class="knowledge-check-link" href="https://edgeguides.rubyonrails.org/working_with_javascript_in_rails.html#an-introduction-to-ajax" target="_blank">Why would you want to use AJAX to load large batches of data?</a>
```

Comment:
Are there other lessons that we use `target="_blank"` in the Knowledge Check links? If *think* this is a departure from our standard pattern, which if so I'm not sure we want to do, at least not in only one lesson.

Reply:
My mistake. I did this out of habit. I do personally think it would benefit the students so that they have another page opened instead of leaving their current page. I do believe this would deviate from the standard you all have set. 

I can remove them.

Reply:
I don't disagree that this might be a beneficial change, at least for external links. I think jump links should stay on the current page. We would probably want to do them across all Knowledge Checks though rather than just this one.

If that's something you'd like to do I can check with the other maintainers and see their thoughts and make a ticket for that if they like the idea as well.